path-traversal/AlexanderLashov_imgapi/app.js

```
@@ -0,0 +1,172 @@
+    const path = require('path');
+
+    req.files.forEach(file => {
+        var metadata = exif.parseSync(file.path).GPSInfo;
+        if (!metadata || !metadata.GPSLatitudeRef || !metadata.GPSLatitude || !metadata.GPSLongitudeRef || !metadata.GPSLongitude) {
+            console.log(`No GPS data found in ${file.path}`);
+            return;
+        }
+        try {
+            var gpsfile = JSON.parse(fs.readFileSync('uploads/GPSInfo.json'));
+        } catch (e) {
+            var gpsfile = {}
```

Biome (2.1.2)

[error] Shouldn't redeclare 'gpsfile'. Consider to delete it or rename it.

'gpsfile' is defined here:


(lint/suspicious/noRedeclare)